src/Shared/Recruit.Vacancies.Client/Infrastructure/Client/QaVacancyClient.cs

```diff
@@ -72,14 +74,41 @@ public Task<Qualifications> GetCandidateQualificationsAsync()
 
         public async Task<QaDashboard> GetDashboardAsync()
         {
-            var dashboard = await _queryStoreReader.GetQaDashboardAsync();
+            var dashboard = await _queryStoreReader.GetQaDashboardAsync().ConfigureAwait(true);
 
             //todo - will be deleted
-            var allReviews = await _reviewRepository.GetActiveAsync();
+            var allReviews = await _reviewRepository.GetActiveAsync().ConfigureAwait(true);
             dashboard.AllReviews = allReviews.ToList();
+
             return dashboard;
         }
 
+        public async Task<List<VacancyReviewSearch>> GetSearchResultsAsync(string searchTerm)
+        {
+            if (TryGetVacancyReference(searchTerm, out var vacancyReference))
+            {
+                return await _reviewRepository.SearchAsync(vacancyReference).ConfigureAwait(false);
+            }
+            return new List<VacancyReviewSearch>();
+        }
+
+        /// <summary>
+        /// This could be somewhere more common, we can refactor when we come across a second use
```

Comment:
This could be a extension method on the `string` class. Best place to put it would be in the shared project so both Employer and QA app can use it. Also this shouldn't be called from the client but by the web projects since the `VAC` prefix is purely a display/front-end concern.

Reply:
the search term is going to be common across multiple fields, so the code is appropriately placed on the client.